src/java.base/unix/native/libjava/io_util_md.c

```diff
@@ -250,7 +253,7 @@ handleGetLength(FD fd)
     if (result < 0) {
         return -1;
     }
-    #ifdef BLKGETSIZE64
+    #if defined(__linux__) && defined(BLKGETSIZE64)
```

Comment:
Looks okay although would be good to fix the indentation at L256 before you integrate.

Reply:
done fixed the indentation